calculations/queries/tree.lp

```diff
@@ -0,0 +1,10 @@
+field(Card, "workflowStateCategory", Category) :-
+    field(Card, "workflowState", State),
+    field(Card, "cardtype", CardType),
+    field(CardType, "workflow", Workflow),
+    workflowState(Workflow, State, Category).
+
+select("title";"workflowStateCategory";"base/fieldtypes/progress";"rank").
```

Comment:
No need to fix this in this PR, but.
This has now a dependency on module "base" and will not work without it. 
We should somehow skip calculations where the base module is not installed. 
Or we should always include base module in the software.

Reply:
The progress is defined to be optional, so it works even if base module hasn't been added I think. Thought I do think, there could be at least some kind of base module, which is tied to cyberismo app, so we can create predefined calculated fields(like progress) which can be used in the queries